### PR TITLE
Update Reith GEL fonts to v2.512 and use different Reith Serif Light font URL

### PR DESCRIPTION
**Pull request by nicola-d-bbc**
Resolves WSTEAMA-64

**Overall change:**
- Updates Reith fonts retrieved from GEL to v2.512
- Changes Reith Serif Light URL to use numbers only subset to decrease file size

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project
- [ ] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/legacy/psammead-styles/src/__snapshots__/fonts.test.js.snap [hunk 1]
```diff
@@ -678,7 +678,7 @@ exports[`Psammead Styles - Fonts should match REITH SERIF LIGHT base font url 1`
 "
   @font-face {
     font-family: \\"ReithSerif\\";
-    src: url(\\"https://gel.files.bbci.co.uk/r2.511/BBCReithSerif_W_Lt.woff2\\") format(\\"woff2\\"), url(\\"https://gel.files.bbci.co.uk/r2.511/BBCReithSerif_W_Lt.woff\\") format(\\"woff\\");
+    src: url(\\"https://gel.files.bbci.co.uk/r2.512/subsets/BBCReithSerif_WNumbers_Lt.woff2\\") format(\\"woff2\\"), url(\\"https://gel.files.bbci.co.uk/r2.512/subsets/BBCReithSerif_WNumbers_Lt\\") format(\\"woff\\");
```
Comment (DarioR01) [inline src/app/legacy/psammead-styles/src/__snapshots__/fonts.test.js.snap]: Is it correct that we are dropping the file extension .woff?

Comment (DarioR01) [reply src/app/legacy/psammead-styles/src/__snapshots__/fonts.test.js.snap]: should it be `...512/subsets/BBCReithSerif_WNumbers_Lt.woff\\` ?

Comment (jroebu14) [reply src/app/legacy/psammead-styles/src/__snapshots__/fonts.test.js.snap]: Good spot!

Comment (nicola-d-bbc) [reply src/app/legacy/psammead-styles/src/__snapshots__/fonts.test.js.snap]: Nice spot!

File: src/app/legacy/psammead-styles/src/__snapshots__/fonts.test.js.snap [hunk 2]
```diff
@@ -688,7 +688,7 @@ exports[`Psammead Styles - Fonts should match REITH SERIF LIGHT with overridden
 "
   @font-face {
     font-family: \\"ReithSerif\\";
-    src: url(\\"https://example.com/BBCReithSerif_W_Lt.woff2\\") format(\\"woff2\\"), url(\\"https://example.com/BBCReithSerif_W_Lt.woff\\") format(\\"woff\\");
+    src: url(\\"https://example.com/subsets/BBCReithSerif_WNumbers_Lt.woff2\\") format(\\"woff2\\"), url(\\"https://example.com/subsets/BBCReithSerif_WNumbers_Lt\\") format(\\"woff\\");
```
Comment (DarioR01) [inline src/app/legacy/psammead-styles/src/__snapshots__/fonts.test.js.snap]: Same here btw

**Review by DarioR01 — APPROVED** [review]
Other than some extension issues LGTM 👍

**Review by jroebu14 — APPROVED** [review]
Thanks for sorting this stuff out @nicola-d-bbc

File: src/app/legacy/psammead-styles/src/__snapshots__/fonts.test.js.snap (hunk 2 above)
Comment (DarioR01) [reply src/app/legacy/psammead-styles/src/__snapshots__/fonts.test.js.snap]: Just realised you pushed the fix for this 👍

**Review by eagerterrier — APPROVED** [review]
T-Shirt-Image-2_1-1

**Pull request merged**